test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Builder/StructuralTypeConfigurationOfTStructuralTypeTest.cs

```diff
@@ -4,6 +4,7 @@
 using Microsoft.AspNet.OData.Builder;
 using Microsoft.AspNet.OData.Test.Common;
 using Moq;
+using System;
```

Comment:
Nit: System using statements should come before other using statements.

Reply:
Fixed. Thank you.